src/style.css

```diff
@@ -0,0 +1,35 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  background-color: #fff;
+  display: flex;
+  justify-content: flex-start;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__star:last-of-type {
+  margin-right: 0;
+}
+
+.stars--1 > .stars__star:not(:nth-child(n + 2)),
+.stars--2 > .stars__star:not(:nth-child(n + 3)),
+.stars--3 > .stars__star:not(:nth-child(n + 4)),
+.stars--4 > .stars__star:not(:nth-child(n + 5)),
+.stars--5 > .stars__star:not(:nth-child(n + 6)) {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
the `width` and `height` properties are already set, same with `background-repeat` and `background-position`, please remove them